src/util/std_code.cpp

```diff
@@ -136,7 +136,7 @@ codet &code_blockt::find_last_statement()
     {
       DATA_INVARIANT(
         last->operands().size() == 1, "label must have one operand");
-      last=&(to_code(last->op0()));
+      last = &(to_code_label(*last).code());
```

Comment:
This also makes the `DATA_INVARIANT` above unnecessary, which is very good.

Reply:
ok